openidc_client/requestsauth.py

```diff
@@ -23,10 +23,10 @@
 
 """Python-Requests AuthBase wrapping OpenIDCClient."""
 
-import requests
+import httpx
 
 
-class OpenIDCClientAuther(requests.auth.AuthBase):
+class OpenIDCClientAuther(httpx.Auth):
```

Comment:
I would prefer it if we keep this one in place, and create a second one for `httpx`. This way we don't break for people that are using requests still.